### ISSUE TITLE
Is it possible to compile NanoVG under XCode?

### ISSUE THREAD

**Issue by ProgrammingLife** [memononen/nanovg#267]
I've included all needed headers but still get following errors. Is there any article or instruction on how to include NanoVG to XCode project without errors?

> .../nanovg/src/nanovg_gl.h:471:32: Use of undeclared identifier 'glGetUniformBlockIndex'
> .../nanovg/src/nanovg_gl.h:667:2: Use of undeclared identifier 'glGenVertexArrays'; did you mean 'glGenVertexArraysAPPLE'?
> .../nanovg/src/nanovg_gl.h:673:2: Use of undeclared identifier 'glUniformBlockBinding'
> .../nanovg/src/nanovg_gl.h:675:16: Use of undeclared identifier 'GL_UNIFORM_BUFFER_OFFSET_ALIGNMENT'
> .../nanovg/src/nanovg_gl.h:929:20: Use of undeclared identifier 'GL_UNIFORM_BUFFER'
> .../nanovg/src/nanovg_gl.h:1108:16: Use of undeclared identifier 'GL_UNIFORM_BUFFER'
> .../nanovg/src/nanovg_gl.h:1109:16: Use of undeclared identifier 'GL_UNIFORM_BUFFER'
> .../nanovg/src/nanovg_gl.h:1114:3: Use of undeclared identifier 'glBindVertexArray'; did you mean 'glBindVertexArrayAPPLE'?

**Comment by memononen** [comment]
Which header you include? Are you using GL2 or GL3?

**Comment by ProgrammingLife** [comment]
I tried these headers:

```
#include <nanovg.h>
#define NANOVG_GL3_IMPLEMENTATION
#include <OpenGL/gl.h>
#include <nanovg_gl.h>
```

OS X 10.11

**Comment by memononen** [comment]
Looks ok. And you have also included the opengl framework?

**Comment by ProgrammingLife** [comment]
Yes, I've included OpenGL.framework to my project. Also I have included SDL2 project into it because I use Oxygine:  https://github.com/oxygine/oxygine-framework
